src/sme_lib/enemy/FireyPetey.cpp

```diff
@@ -353,26 +320,23 @@ bool TNerveFPFireBreath::execute(TSpineBase<TLiveActor> *spine) const
 
         }
         if(found){
-          mFire->mVelocity.x = 50.0f * sinf(target->mRotation.y);
+          mFire->mVelocity.x = 50.0f * sinf(SME::Util::Math::angleToRadians(target->mRotation.y));
```

Comment:
I'd rather we put `using namespace SME::Util;` at the top of the file and trim out the usage of the `SME::Util` part of the namespace. This also goes for the other instances in these files.